### PR TITLE
dev-games/ogre: Bump to 13.6.1, remove ogre-next

### PR DESCRIPTION
**Pull request by Yamakuzure**
* Bump Stable OGRE to the current stable release v13.6.0
* Remove OGRE-next ebuild v2.1

OGRE-next is already so much different from classic OGRE, that it makes much more sense to give it its own category dev-games/ogre-next at some point. Just not yet.

Bug: https://bugs.gentoo.org/885733
Closes: https://bugs.gentoo.org/885733
Signed-off-by: Sven Eden <sven.eden@prydeworx.com>

### CODE REVIEW

**Comment by Yamakuzure** [comment]
While waiting, 13.6.1 has been released.

File: dev-games/ogre/ogre-13.6.1.ebuild [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```
Comment (juippis) [inline dev-games/ogre/ogre-13.6.1.ebuild:4]: Could you use the latest available EAPI here, 8?

**Review by juippis — COMMENTED** [review]
Hey, 

could you split removing and adding new version to their separate commits? Reviewing the ebuild like this is impossible.

File: dev-games/ogre/ogre-13.6.1.ebuild (hunk 1 above)
Comment (Yamakuzure) [reply dev-games/ogre/ogre-13.6.1.ebuild:4]: Sure, will do

**Comment by Yamakuzure** [comment]
> Hey,
> 
> could you split removing and adding new version to their separate commits? Reviewing the ebuild like this is impossible.

Yes, sorry about that.

**Review by juippis — APPROVED** [review]
Thanks!

**Comment by juippis** [comment]
It looks the old version(s?) cannot be dropped as the tree is pretty messy when it comes to ogre. Revdeps and profile masks here and there.

**Comment by Yamakuzure** [comment]
> It looks the old version(s?) cannot be dropped as the tree is pretty messy when it comes to ogre. Revdeps and profile masks here and there.

_eugh_ ... but at least the new version is there so people can do some testing.

Thank you very much!

**Pull request closed without merging**